boards/arm/blufdali_nrf52840/Kconfig

```
@@ -0,0 +1,9 @@
+# dalibluf nRF52840 board configuration
+
+# Copyright (c) 2021 innblue
+
+config BOARD_ENABLE_DCDC
+	bool "Enable DCDC mode"
```

As per the guidelines, please remove `Enable`:
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Maybe just write:
```suggestion
	bool "DCDC mode"
```